flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -409,6 +409,18 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
                             "org.apache.flink.table.runtime.functions.scalar.ArrayMinFunction")
                     .build();
 
+    public static final BuiltInFunctionDefinition SPLIT =
+            BuiltInFunctionDefinition.newBuilder()
+                    .name("SPLIT")
+                    .kind(SCALAR)
+                    .inputTypeStrategy(
+                            sequence(
+                                    logical(LogicalTypeFamily.CHARACTER_STRING),
+                                    logical(LogicalTypeFamily.CHARACTER_STRING)))
+                    .outputTypeStrategy(forceNullable(explicit(DataTypes.ARRAY(STRING()))))
```

Comment:
I think this `outputTypeStrategy` does not follow the description.

The description says output can be `null` only if any of the input parameters are `null`. If both arguments are `NOT NULL` the result is also `NOT NULL`.

Reply:
Ok, I use `nullableIfArgs(explicit(DataTypes.ARRAY(STRING())))` now.